src/screens/ProfileScreen/index.js

```diff
@@ -273,9 +276,6 @@ const ProfileScreen = ({route}) => {
       const result = await getSelfFeedsInProfile(offset, limit);
       const {data: dataMyFeed} = result;
       const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
-      if (Array.isArray(result.data) && result.data.length === 0) {
-        setIsLastPage(true);
-      }
       if (offset === 0) {
         StorageUtils.myFeeds.set(JSON.stringify(mapNewData));
         setMyProfileFeed(mapNewData, myProfileDispatch);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the old hunk, there was a check to see if the result data array length was 0, and if so, `isLastPage` was set to true. This logic has been removed in the new hunk. If there's no other mechanism to handle the situation where there are no more feeds to fetch, this could potentially lead to an issue where the app continuously tries to fetch more feeds even though there are none left.



<!-- This is an auto-generated comment by OSS CodeRabbit -->